controllers/grid/users/stageParticipant/StageParticipantGridHandler.inc.php

```diff
@@ -115,6 +115,32 @@ function initialize($request) {
 			$cellProvider
 		));
 
+		if (Validation::isLoggedInAs() && isset($this->getRequestArgs()['submissionId']) && !empty( $this->getRequestArgs()['submissionId'] ) ) 
```

Comment:
I think you should be able to fetch the submission ID more safely (post-validation) using e.g.:
```
$submission = $this->getSubmission();
$submissionId = $submission->getId();
```